app/src/main/java/com/infomaniak/mail/utils/MessageBodyUtils.kt

```diff
@@ -51,27 +52,32 @@ object MessageBodyUtils {
         "[name=\"quote\"]", // GMX
     )
 
-    fun splitContentAndQuote(body: Body): SplitBody {
+    fun splitContentAndQuote(body: Body): SplitBody = runBlocking(Dispatchers.IO) {
```

Comment:
@sirambd Do we want to use `Dispatchers.IO` here, or some injection or something else ?

Reply:
There's no api call here, no file read/write, so there's no need for Dispatchers.IO